src/history.md

```diff
@@ -0,0 +1,73 @@
+# History
+
+## BIOS
+
+BIOS is the good old, inscrutable, unstructured, non-standard way of
+initializing a hardware platform in the pre-UEFI days. In other words it's a
+binary blob with no standardized structure, that is responsible for
+initializing CPU and memory, and jumping to a hard-coded position on the MBR of
+the first disk drive.
+
+BIOS has been largely replaced by the (much better) UEFI over the past 20
+years. Many UEFI implementations still offer a "BIOS compatibility mode" which
+make it behave like an old BIOS, with its (lack of) features.
+
+BIOS is also a misused term for system firmware nowadays. You may still hear
+"BIOS" in reference to system firmware, either it's UEFI or even LinuxBoot.
+However, "BIOS" refers to a specific type of firmware, and UEFI is definitely
+not BIOS, just like LinuxBoot is not BIOS.
+
+## LinuxBIOS
+
+The `LinuxBIOS` project was created in 1999 by Ron Minnich, Stefan Reinauer and
+others. It is not much younger than UEFI, but they were already experimenting
+the idea of running Linux as firmware! Like many great ideas, it was way ahead
```

Comment:
Let's drop the exclamation mark `!` here.

Reply:
note to self:
https://web.archive.org/web/20070430170020/http://www.linuxbios.org/Welcome_to_LinuxBIOS

Reply:
see #105